packages/nextjs/components/impact-vector/ImpactVectorGraph.tsx

```diff
@@ -123,7 +123,7 @@ export default function ImpactVectorGraph({
         <ComposedChart
           width={500}
           height={300}
-          data={[...transformedData].splice(0, projectsShown)}
+          data={transformedData.slice(-projectsShown, -1)}
```

Comment:
Is there a reason for shaving the top project off with the `-1`? If not, I think we could grab all of the highest projects with `.slice(-projectsShown)`

Reply:
Good catch! Commit incoming.